client/src/components/event-types/EventTypesList.vue

```diff
@@ -128,3 +136,8 @@ export default {
     }
 };
 </script>
+<style>
+.search-input .v-input__slot {
+    background: rgba(0, 0, 0, 0.12);
```

Comment:
color hard coded and not taken form styles

Reply:
Still actual